### DISCUSSION TITLE
mediawiki compilation

### DISCUSSION THREAD

**Discussion by obujeiromhginsurance** [peachpiecompiler/peachpie#1091] [Ideas]
Has any one been able to compile mediawiki's latest version with .Net 5.0 or .Net 6.0?
I tried and it never ended.

Thanks

**Comment by bfistein** [comment]
Hi @obujeiromhginsurance. Very unlikely, there's almost certainly a bug on our side. We are aware of [the issue you posted](https://github.com/peachpiecompiler/peachpie/issues/1081), but haven't been able to check it yet.
